modules/swagger-codegen/src/main/resources/go/api_client.mustache

```diff
@@ -2,6 +2,11 @@ package {{packageName}}
 
 import (
     "strings"
+    "github.com/go-resty/resty"
+    "errors"
+    "reflect"
+    "bytes"
+    "path/filepath"
 )
 
 type ApiClient struct {
```

Comment:
This should be spelled `APIClient`  per the usual [Go conventions](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).